src/server.c

```diff
@@ -1056,7 +1056,9 @@ void databasesCron(void) {
      * as primary will synthesize DELs for us. */
     if (server.active_expire_enabled) {
         if (iAmPrimary()) {
-            activeExpireCycle(ACTIVE_EXPIRE_CYCLE_SLOW);
+            if (!server.import_mode) {
```

Comment:
any reason for a nested if here?

Reply:
I guess the reason is the `else` below.

Reply:
It could be rewritten as this though, if that's better?

```C
if (!iAmPrimary()) {
    expireReplicaKeys();
} else if (!server.import_mode) {
    activeExpireCycle(ACTIVE_EXPIRE_CYCLE_SLOW);
}

Reply:
The reason is just as @zuiderkwast said. it's a bit strange and I have fixed it. Thanks @zuiderkwast.